ui/settingsdlg.ui

```diff
@@ -129,16 +129,16 @@
              </widget>
             </item>
             <item row="0" column="1">
-             <widget class="QSpinBox" name="max_priv_storage">
+             <widget class="SizeSpinBox" name="max_priv_storage">
               <property name="enabled">
                <bool>true</bool>
               </property>
+              <property name="toolTip">
+               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Values displayed using the binary definition of gigabyte and terabyte, i.e. 1024&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt; and 1024&lt;span style=&quot; vertical-align:super;&quot;&gt;4 &lt;/span&gt;bytes respectively.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
gigabyte and megabyte (gibibyte and mebibyte?)

Reply:
I'm confused to be honest. What do you mean by that?